akka-docs/src/test/java/jdocs/stream/operators/SinkDocExamples.java

```diff
@@ -6,122 +6,126 @@
 
 import akka.NotUsed;
 import akka.actor.ActorSystem;
-
-import akka.stream.javadsl.Flow;
+import akka.japi.Pair;
```

Comment:
It would be better not to rearrange the imports, since now the 'Pair' import is no longer part of the `#takeLast-operator-example` snippet.

Reply:
Okay I'll resolve this